examples/restore_dashboards.py

```diff
@@ -13,18 +13,33 @@
 #
 # Parse arguments
 #
-if len(sys.argv) != 3:
-    print 'usage: %s <sysdig-token> <file-name>' % sys.argv[0]
+if len(sys.argv) <= 2:
+    print 'usage: %s <sysdig-token> <file-name> <backend_url> <ssl_check>' % sys.argv[0]
     print 'You can find your token at https://app.sysdigcloud.com/#/settings/user'
+    print 'Optional: backend_url (https://app.sysdigcloud.com, https://app-staging.sysdigcloud.com)'
+    print 'Optional: ssl_check (True/False)'
     sys.exit(1)
 
 sdc_token = sys.argv[1]
 dashboard_state_file = sys.argv[2]
 
+try:
+    backend_url = sys.argv[3]
+except:
+    backend_url = 'https://app.sysdigcloud.com'
+
+try:
+    if (sys.argv[4] == "False"):
+        ssl_check = False
+    else:
+        ssl_check = True
+except:
+    ssl_check = True
```

Comment:
These try/except don't look well (I believe PEP8 recommends avoiding this use of try/except) . I would use argparse, but at least we could clean this out with a:
```python
ssl_check = len(sys.argv) > 4 and sys.argv[4] == 'True'
# or for the previous one (which would still be wrong because of my previous comment)
bakend_url = sys.argv[3] if len(sys.argv) > 3 else 'https://app.sysdigcloud.com'
```